amr-wind/physics/multiphase/MultiPhase.cpp

```diff
@@ -10,16 +10,17 @@ MultiPhase::MultiPhase(CFDSim& sim)
     , m_mueff(sim.pde_manager().icns().fields().mueff)
     , m_density(sim.repo().get_field("density"))
 {
-    // Register levelset equation
-    auto& levelset_eqn = sim.pde_manager().register_transport_pde("Levelset");
-    // Defer getting levelset field until PDE has been registered
-    m_levelset = &(levelset_eqn.fields().field);
-
     amrex::ParmParse pp_multiphase("MultiPhase");
     pp_multiphase.query("density_fluid1", m_rho1);
     pp_multiphase.query("density_fluid2", m_rho2);
     pp_multiphase.query("viscosity_fluid1", m_mu1);
     pp_multiphase.query("viscosity_fluid2", m_mu2);
```

Comment:
Use transport model here.

Reply:
Two phase transport model is implemented + tested